en/clubs/mottagningen/sidebar.md

```diff
@@ -3,50 +3,56 @@
 The individuals responsible for the Reception are referred to as Title. If you would like to contact us, you can send an email to [titel@datasektionen.se](mailto:titel@datasektionen.se). If you would like to contact a specific person, their contact information can be found below.
 
 ### Det Kongliga Dadderiet 
-**Storasyster**, <br />
-Ebba Bråtman<br />
-[storasyskon@datasektionen.se](mailto:storasyskon@datasektionen.se)<br />
-076-192 70 22
-
-**Halvbror**, <br />
+**Storebror**, <br />
 Carl Hansson<br />
-[halvbror@datasektionen.se](mailto:halvbror@datasektionen.se)<br />
+[storasyskon@datasektionen.se](mailto:storasyskon@datasektionen.se)<br />
 073-832 82 13
 
+**Halvsyster**, <br />
+Meya Wikner<br />
+[halvsyster@datasektionen.se](mailto:halvsyster@datasektionen.se)<br />
+072-200 48 83
+
 **Lillebror**, <br />
-Henry Lou<br />
+Christoffer Andersson<br />
 [lillebror@datasektionen.se](mailto:lillebror@datasektionen.se)<br /> 
-072-423 62 97
+076-177 72 77
 
 **Lillasyster**, <br />
-Lovisa Sjöberg <br />
+Tilde Tärnvik <br />
 [lillasyster@datasektionen.se](mailto:lillasyster@datasektionen.se)<br /> 
-070-364 47 79
+070-672 07 89
 
 ### Det Kongliga Drifveriet
 **Konglig Öfverdrif**, <br />
-Bror Sebastian Sjövald<br />
+Lucas Dow<br />
 [ofverdrif@datasektionen.se](mailto:ofverdrif@datasektionen.se)<br />
-073-600 70 21
+073-551 00 01
 
 **Konglig Direktifdrif**, <br />
-Lucas Dow<br />
+Elin Andersson Lundell<br />
 [direktifdrif@datasektionen.se](mailto:direktifdrif@datasektionen.se)<br />
-073-551 00 01
+072-164 28 12
 
 **Konglig Indrif**, <br />
-Olivia Aronsson<br />
+Lydia Brorsson<br />
 [indrif@datasektionen.se](mailto:indrif@datasektionen.se)<br />
-070-685 87 74
+073-528 78 63
 
 ### Det Kongliga Doqumenteriet
 **Samdoquise**, <br />
-Eric Sandberg<br />
+Markus Viderfors<br />
```

Comment:
I think this is wrong? I assume Markus is not both Storquisine and Samdoquise?

Reply:
Yeah you´re right!